content/kubernetes/re-clusters/upgrade-redis-cluster.md

```diff
@@ -78,62 +83,96 @@ customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redis
+        repository:       redislabs/redis
+        versionTag:       <new-version-tag>
+    ```
 
-## How does the upgrade work?
+1. Save the changes to apply.
 
-Rolling updates allow you to update deployments with zero downtime
-by incrementally updating Redis Enterprise Cluster instances in the pods with new instances.
+### Monitor the upgrade
 
-These diagrams show how a rolling update works:
+You can view the state of the REC with `kubectl get rec`.
 
-- Each hexagon represents a node
-- Each box represents a pod
+  During the upgrade, the state should be `Upgrade`.
+  When the upgrade is complete and the cluster is ready to use, the state will change to `Running`.
+  If the state is `InvalidUpgrade`, there is an error (usually relating to configuration) in the upgrade.
+
+```sh
+$ kubectl get rec
+NAME   NODES   VERSION      STATE     SPEC STATUS   LICENSE STATE   SHARDS LIMIT   LICENSE EXPIRATION DATE   AGE
+rec    3       6.2.10-107   Upgrade   Valid         Valid           4              2022-07-16T13:59:00Z      92m
+```
+
+To see the status of the current rolling upgrade, run:
+
+```sh
+kubectl rollout status sts <REC_name>
+```
 
-![kubernetes-rolling-updates](/images/rs/kubernetes-rolling-updates.png)
+### Upgrade databases
 
-The pods are updated one by one in the diagram, starting from left to right.
-Each pod is updated after the last one completes successfully.
+After the cluster is upgraded, you can upgrade your databases. The process for upgrading databases is the same for both Kubernetes and non-Kubernetes deployments. More details on how to [upgrade a database]({{<relref "/rs/installing-upgrading/upgrading#upgrade-a-database">}}) in the [Upgrade an existing Redis Enterprise Software deployment]({{<relref "/rs/installing-upgrading/upgrading.md">}}) documentation.
 
-![kubernetes-rolling-updates-newapp](/images/rs/kubernetes-rolling-updates-newapp.png)
+Note that if your cluster [`redisUpgradePolicy]({{<relref "/kubernetes/reference/cluster-options#redisupgradepolicy">}}) or your database [`redisVersion`]({{<relref "">}}) are set to `major`, you won't be able to upgrade those databases to minor versions. See [Redis upgrade policy]({{<relref "/rs/installing-upgrading/upgrading#redis-upgrade-policy">}}) for more details.
 
-![kubernetes-rolling-updates-newcluster](/images/rs/kubernetes-rolling-updates-newcluster.png)
+## How does the REC upgrade work?
 
-![kubernetes-rolling-updates-done](/images/rs/kubernetes-rolling-updates-done.png)
+The Redis Enterprise cluster (REC) uses a rolling upgrade, meaning it upgrades pods one by one. Each pod is updated after the last one completes successfully. This helps keep your cluster available for use.
 
-The pods in the StatefulSet are updated in reverse ordinal order.
-The Kubernetes controller terminates each pod and waits for it to transition to `Running` and then to `Ready`.
+To upgrade, the cluster terminates each pod and deploys a new pod based off the new image.
```

Comment:
I'm not a grammar authority by any means, but is "based off" correct here, or should it be "based on"?